samples/ambient-argo/tag-chart/templates/shimservice.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if ((.Values.base.tags.default).revision) }}
```

Comment:
You explained this to me earlier but might be useful to add a comment on why this is needed (unless it's already in another doc somewhere)

Reply:
good point.  For posterity, this is due to a missing feature in ztunnel in 1.18 and 1.19 where it didn't respect revisions, but always send XDS requests to the istiod service.  This shim creates an istiod service, so that you can point the ztunnel at whatever control plane you need.  This is fixed in 1.20.